src/Game.js

```diff
@@ -1,31 +1,38 @@
+
 import Wheel from './Wheel.js';
 import Game from './Game.js';
 import Player from './Player.js';
 import Puzzle from './Puzzle.js';
 import Round from './Round.js';
 import data from './data/sample-data';
 
+
 class Game {
-  constructor(p1, p2, p3) {
-    this.players = [];
+  constructor(wheel) {
     this.roundCounter = 0;
-    this.player1 = new Player(1, p1);
-    this.player2 = new Player(2, p2);
-    this.player3 = new Player(3, p3);
-    this.round = this.makeNewRound();
-    this.wheel = new Wheel(); 
+    this.players = [];
+    this.round;
```

Comment:
Let’s give this a default value? What’s it going to represent? The round number? And object of round details? Always default your instance properties to an appropriate data type.